static/js/publisher/form.js

```diff
@@ -1,6 +1,7 @@
-import { initSnapScreenshotsEdit } from './screenshots';
+import { initSnapScreenshotsEdit } from './market/screenshots';
```

Comment:
This file got quite confusing now. While it has some bits that are common to both forms it has also a lot of things for one or another - like unnecessary screenshots on admin page or unnecessary whitelist multiselect on market page.

But let's handle it in the refactor task.